Tests/Entity/SnapshotManagerTest.php

```diff
@@ -18,6 +18,78 @@
  */
 class SnapshotManagerTest extends \PHPUnit_Framework_TestCase
 {
+    public function testSetTemplates()
+    {
+        $manager = $this->getMockBuilder('Sonata\PageBundle\Entity\SnapshotManager')
+            // this method is called in SonataPageExtension
+            ->setMethods(array(
+                'setTemplates'
```

Comment:
What does this do ? If this mocks setTemplates, then the logic in `setTemplates` will never be called and the test cannot work, right ?